x11-misc/blockdpy/blockdpy-1-r1.ebuild

```diff
@@ -20,19 +21,21 @@ RDEPEND="${DEPEND}"
 S=${WORKDIR}
 
 src_unpack() {
-	cp "${DISTDIR}"/blockdpy.c blockdpy.c
+	cp "${DISTDIR}"/blockdpy.c blockdpy.c || die
+}
 
+src_prepare() {
 	# Add includes to avoid QA warnings.
-	sed -i '/#include <stdio.h>/i#include <stdlib.h>' blockdpy.c
-	sed -i '/#include <stdio.h>/i#include <string.h>' blockdpy.c
+	sed -i '/#include <stdio.h>/i#include <stdlib.h>' blockdpy.c || die
+	sed -i '/#include <stdio.h>/i#include <string.h>' blockdpy.c || die
```

Comment:
have you tested this code? I don't believe this can work... Please always test your ebuilds